src/BaseSchema.hack

```diff
@@ -53,5 +53,9 @@ abstract class BaseSchema implements Introspection\__Schema {
         return $type is nonnull ? $type::nonNullable()->nullableForIntrospection() : null;
     }
 
-    // TODO add method to create singleton
+    <<__Override>>
+    final public function getTypes(): vec<Introspection\__Type> {
+        // TODO: should we filter out private types? ie. anything with __
```

Comment:
Yes, the spec says those should not show up in introspection. The spec also requires that the `__` prefix is not used for anything else (although we're not enforcing it right now), so this check should be safe.